crates/basilica-sdk-python/tests/test_async_methods.py

```
@@ -0,0 +1,441 @@
+        deployment = self._create_mock_deployment()
+        ready_status = self._create_ready_status()
+
+        mock_response = MagicMock()
+        mock_response.state = "Active"
+        mock_response.replicas.ready = 1
+        mock_response.replicas.desired = 1
+        mock_response.progress = None
+        deployment._client.get_deployment = MagicMock(return_value=mock_response)
+
+        asyncio_sleep_called = False
+        original_sleep = asyncio.sleep
```

Ruff (0.14.11)

Local variable `original_sleep` is assigned to but never used

Remove assignment to unused variable `original_sleep`

(F841)